app/src/common/components/section-list/index.jsx

```diff
@@ -16,66 +13,78 @@ import i18n from 'i18next';
         connected: state.login.isConnected
     }),
     dispatch => ({
-        clearSection: () => dispatch(clearSection()),
         loadSectionList: () => dispatch(loadSectionList()),
-        loadArticleList: () => dispatch(loadArticleList()),
-        loadSection: (id) => dispatch(loadSection(id)),
-        getArticles: (sectionId) => dispatch(getArticles(sectionId))
+        getArticles: (sectionId) => dispatch(getArticles(sectionId)),
+        search: () => dispatch(searchArticleList())
     })
 )
 export class SectionList extends React.Component {
 
     componentWillMount() {
         this.props.loadSectionList();
-        this.props.clearSection();
     }
 
-    componentDidUpdate() {
-        this.setAllSectionListRender();
-        if (this.props.sectionID) {
-            this.props.getArticles(+this.props.sectionID);
-        } else {
-            this.props.loadArticleList();
+    renderSectionList = () => {
+        const {sections} = this.props;
+        return (
+            <div>
+                <button className="accordion" onClick={() => this.onClickHandler(null, 0) } ref={`button${0}`}>Tous les articles</button>
+                <div className="panel">
+                    <ArticleList />
+                </div>
+                {sections && sections.length > 0 ?
+                    sections.map((section, index) => {
+                        return (
+                            <div>
+                                <button className="accordion" onClick={() => this.onClickHandler(section.id, index+1) } ref={`button${index+1}`}>{section.name}</button>
+                                <div className="panel">
+                                    <ArticleList />
+                                </div>
+                            </div>
+                        );
+                    }) : null
+                }
+            </div>
+        );
+    };
```

Comment:
They got this : http://www.material-ui.com/#/components/list

I wanted to fit with this : https://support.google.com/chrome/?hl=fr#topic= @JabX